api/server/router/container/copy.go

```diff
@@ -98,7 +101,26 @@ func (s *containerRouter) getContainersArchive(ctx context.Context, w http.Respo
 	}
 
 	w.Header().Set("Content-Type", "application/x-tar")
```

Comment:
[This](https://superuser.com/a/960710) suggests that maybe the content-type should change as well?

Reply:
I think since the `Content-Encoding` header is being set to `gzip` the `Content-Type` should stay as `application/x-tar` as it is the underlying content type after decoding:

> The `Content-Encoding` entity header is used to compress the media-type. When present, its value indicates which encodings were applied to the entity-body. *It lets the client know, how to decode in order to obtain the media-type referenced by the `Content-Type` header.*

https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Encoding